modules/.submodules.json

```diff
@@ -0,0 +1,12 @@
+{
+  "userId": [
+    "digiTrustIdSystem",
+    "pubCommonIdSystem",
+    "unifiedIdSystem",
+    "id5IdSystem",
+    "unifiedIdSystem"
```

Comment:
I think someone handled a merge improperly here.  Duplicate and removed modules are showing up now...

Might want to check to make sure no other changes were accidentally reverted as well.

Reply:
Yeah this is supposed to be:
```
{
  "userId": [
    "digiTrustIdSystem",
    "id5IdSystem"
],
  "adpod": [
    "freeWheelAdserverVideo"
  ]
}
```